server/index.ts

```diff
@@ -8,7 +8,7 @@ import UserService from './services/userService'
 
 promClient.collectDefaultMetrics()
 
-const hmppsAuthClient = new HmppsAuthClient(new TokenStore(createRedisClient()))
+const hmppsAuthClient = new HmppsAuthClient(new TokenStore(createRedisClient({ legacyMode: false })))
```

Comment:
should this be `true` instead?

Reply:
want this to be false so can use async api